routes/product.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { createProduct } from '../controllers/product.js';
+import { getProducts } from '../controllers/product.js';
+import { getProduct } from '../controllers/product.js';
+const router = express.Router();
+
+router.route('/').post(createProduct);
+router.route('/:id').get(getProducts);
```

Comment:
getProducts is get all the products. You don't need an id here. you can chain it to line 7 as .get(getProducts)